### ISSUE TITLE
Issue with number of units for softmax layer

### ISSUE THREAD

**Issue by geconf** [kalaspuffar/tensorflow-data#8]
Hello @kalaspuffar,

Thank you very much for this example, your video was very helpful to me. I believe there is a typo in line 44 ` num_classes = 3` (should be 2 as there are only two categories).

Best,
Gerardo

**Comment by kalaspuffar** [comment]
Hi @GerardoConsuelos 

I think you might be right there.

To my defence, I worked a lot at object detection at the time and there you have the classes plus background as the last category. But in Object recognition, you don't need the background type.

Best regards
Daniel
